app/Services/PostGIS.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Services;
+
+class PostGIS extends BaseService
```

Comment:
Thanks for your PR! Would you please run PHPCS and Tlint on it and then submit again? Thanks!

Reply:
Sure thing. How much of the PHP DocBlock errors from PHPCS do you need?

Reply:
Ya know, this is a good point--we didn't have PHPCS set up here.

I've now set it up; would you mind merging `main` into this branch and then running `./vendor/bin/phpcs` and checking for any warnings or errors in this file?

Reply:
Done! Looks like we're good.